packages/api/endpoints/rules.js

```diff
@@ -60,26 +64,28 @@ async function get(req, res) {
  * @returns {Promise<Object>} the promise of express response object
  */
 async function post(req, res) {
+  const {
+    model = new models.Rule(),
+    dbClient = await getKnexClient(),
+  } = req.testContext || {};
+
+  let record;
+  const rule = req.body || {};
+  const name = rule.name;
+
+  if (await model.exists(name)) {
+    return res.boom.conflict(`A record already exists for ${name}`);
+  }
+
+  const postgresRecord = await translateApiRuleToPostgresRule(rule, dbClient);
+
   try {
-    const data = req.body;
-    const name = data.name;
-
-    const model = new models.Rule();
-
-    try {
-      await model.get({ name });
-      return res.boom.conflict(`A record already exists for ${name}`);
-    } catch (error) {
-      if (error instanceof RecordDoesNotExist) {
-        const record = await model.create(data);
-
-        if (inTestMode()) {
-          await addToLocalES(record, indexRule);
-        }
-        return res.send({ message: 'Record saved', record });
-      }
-      throw error;
-    }
+    await dbClient.transaction(async (trx) => {
+      await trx(tableNames.rules).insert(postgresRecord, 'cumulus_id');
+      record = await model.create(rule, rule.createdAt, rule.updatedAt);
```

Comment:
`rule.createdAt` is no longer set, so I think this value is undefined and thus the rule is not being created with the same values. I see now why you were setting `rule.createdAt` before `translateApiRuleToPostgresRule`, so we can restore that if you want so that the records are in sync. Sorry for misunderstanding before

Reply:
Yeah, that is correct. Sorry I didn't do a great job of explaining my reasoning before. I've since updated it.